pkg/reconciler/testing/logger.go

```diff
@@ -31,6 +31,16 @@ func SetupFakeContext(t *testing.T) (context.Context, []controller.Informer) {
 	return WithLogger(ctx, t), informer
 }
 
+// SetupDefaultContext sets up the the Context and the default filtered informers for the tests.
+func SetupDefaultContext(t *testing.T) (context.Context, []controller.Informer) {
```

Comment:
same as `SetupFakeContext`:

https://github.com/tektoncd/pipeline/blob/00b14d78ad73c055451f2a714387cd03f6222b89/pkg/reconciler/testing/logger.go#L25-L32

Reply:
Removed `cloudEventClientBehaviour` as not needed but this is to call the above `setupDefaultContextWithLabelKey` used to test spire with the fakeworkloadAPI.